specs/binding-kafka.spec/src/main/resources/META-INF/zilla/kafka.idl

```diff
@@ -438,14 +471,16 @@ scope kafka
 
         struct KafkaOffsetCommitBeginEx
         {
-            string16 groupId;
             string16 topic;
+            string16 groupId;
+            int32 generationId;
+            string16 memberId;
         }
 
         struct KafkaOffsetCommitDataEx
         {
-            int32 partitionId;
-            int64 partitionOffset;
+            KafkaOffsetCommitted partition;
```

Comment:
If metadata is `DATA` payload, then we just need `KafkaOffset` here instead of `KafkaOffsetCommitted`, agree?